_src/docs.css

```
@@ -777,6 +777,16 @@ img {
   display: inline-block;
 }
 
+div.article > * >  img {
+  cursor: zoom-in;
+  transition: transform ease-in-out 0.5s;
+} 
+
+div.article > * >  img.zoomed {
+  cursor: zoom-out;
+  transform: scale(2.5);
```

```suggestion
  transform: scale(1.75);
```
Is there a way to limit the scaled width to the current with of the window? I looked around but didn't find a simple option. If not, 1.75 or 2.0 seems to work better. 2.5 is too large.